docs/development-and-gardener-setup.md

```diff
@@ -72,14 +14,15 @@ eval $(minikube docker-env)
 make container
 ```
 
-Follow the installation instructions above.
+Follow the [installation instructions](/install/README.md).
 
 After that, you can repeat the following commands to rebuild and test changes:
 
 ```
 kubectl delete -n kube-system deployment karydia || true
 eval $(minikube docker-env)
 make container
+helm template ./install/charts/ --output-dir manifests -x ./templates/deployment.yaml 
 cat manifests/deployment.yml | sed -e 's|image: eu.gcr.io/gardener-project/karydia/karydia.*|image: eu.gcr.io/gardener-project/karydia/karydia|' | kubectl apply -f -
```

Comment:
I think this won't work anymore because of new installation method or am I wrong?

Reply:
This is still possible. The `helm template` command will create a usable .yaml file. However, the last line can be changed, as in #120 we changed the used image and did not use a version number anymore.